sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -13,169 +13,176 @@
 #include <CL/sycl/detail/common.hpp>
 #include <CL/sycl/detail/os_util.hpp>
 #include <CL/sycl/detail/pi.h>
+#include <cassert>
+#include <string>
+
+// Function to load the shared library
+// Implementation is OS dependent.
+void *loadOsLibrary(const std::string &library);
```

Comment:
Please, try to use the consistent coding style: for new files - LLVM coding style, for existing ones - the same that is already used in the file. 
Specifically, here "library" on line 25 "Library".